### PR TITLE
Added Rss-news parser

### PR DESCRIPTION
**Pull request by ENDER-9**
Commands

**Add new url**
`$news -add-url <your_rss_feed_url>`
**Delete url**
`$news -delete-url <your_rss_feed_url>`
**Show active urls**
`$news -show-urls`
**Set default channel id**
`$news -set-default-channel-id <new_channel_id>`
**Set channel id for specific url**
`$news -set-specific-channel-id <url> <channel_id>`
**show this message**
`$news --help`
Befor adding your first rss url please make sure you have set the default channel id
Screenshot 2020-08-26 15:33:21
Screenshot 2020-08-26 15:36:35

### CODE REVIEW

**Comment by Tracer1337** [comment]
Heyho my soulmate,
one thing that annoys me is, that there is this ugly dash in front of all commands. ``$news -add-url`` Please remove this

**Comment by ENDER-9** [comment]
Heyho back my dear soulmate,
I have fixi dixied your problemo. Thanks for erinnering me to das

**Comment by Tracer1337** [comment]
Do not hardcode your help embed. Use the discord.py integrated, dynamic one (``<PREFIX>help``)

File: news-links.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (uvulpos) [inline news-links.json:1]: move to ~/.hellocoding-discord-bot NEVER EVER NEVER write user data into the git folder ALWAYS use user settings folder. Should already exists on your machine -> https://github.com/hellocodingDE/hellocoding-bot/blob/master/main.py#L59

**Pull request closed without merging**